app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,463 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
```

Comment:
Hey @OscarFMdev 
What the previous reviewer meant by improving the layout according to the wireframe is try to match the wireframe's position of elements on the page. That includes margins, alignment and position of items.

For example, on the foods page:

- The `add food` button should be aligned right not centre
- Reduce the right and left margins or increase the width of the table

![image](https://user-images.githubusercontent.com/38088818/218197365-7ebcd3a1-1c09-44cb-a6d3-1b625fff2abb.png)

On the Recipe details page:

- The table must be the same width as the text 
- Reduce the right and left margins or increase the width of the recipe details container

![image](https://user-images.githubusercontent.com/38088818/218198784-4f472aee-cb6f-4f92-a265-d088fcf774c2.png)

On the Recipes page:

![image](https://user-images.githubusercontent.com/38088818/218199328-36a79c07-99c4-41f7-a31a-3f9a11c109ed.png)

Same thing on the Public Recipes card:

![image](https://user-images.githubusercontent.com/38088818/218199634-ae353594-6259-4d7f-9e15-9b518b1804b5.png)

Kindly do your best to improve the layout (alignment and position of items) to be as close to the wireframe as possible. Otherwise fantastic job on your application!!! 👏🏾

Reply:
- For the first, understand we were giving the flexibility to have the deign as best fit, and I believe we did follow the wireframe. 
-For the second, the test is short that's why it's not fitted. Inputting more text just as the previous recover did, should show you the difference

Reply:
@Shedrack-Sunday 

- I believe the layout should be the same regardless of whether the text is long or short. It is the duty of a good programmer to make sure that is the case
- To confirm, when you put your application side by side with the wireframe do you see a perfect replica? and you believe that there's absolutely nothing that you can improve about your layout so that you application is the best it could be?

Reply:
@KDlamin thank you for taking the time for explaining and drawing lines so we could understand better, about the question, no, we didn't match entirely the wireframe, we will work on it.